ffi/input_pocketbook.lua

```diff
@@ -0,0 +1,147 @@
+local ffi = require("ffi")
+local C = ffi.C
+local inkview = ffi.load("inkview")
+local util = require("ffi/util")
+require("ffi/linux_input_h")
+
+-- luacheck: push
+-- luacheck: ignore
+local EVT_INIT = 21
+local EVT_EXIT = 22
+local EVT_SHOW = 23
+local EVT_REPAINT = 23
+local EVT_HIDE = 24
+local EVT_KEYDOWN = 25
+local EVT_KEYPRESS = 25
+local EVT_KEYUP = 26
+local EVT_KEYRELEASE = 26
+local EVT_KEYREPEAT = 28
+local EVT_POINTERUP = 29
+local EVT_POINTERDOWN = 30
+local EVT_POINTERMOVE = 31
+local EVT_ORIENTATION = 32
+local EVT_MTSYNC = 39
+local EVT_FOREGROUND = 151
+local EVT_BACKGROUND = 152
+
+local PANEL_DISABLED = 0
+-- luacheck: pop
+
+ffi.cdef [[
```

Comment:
Same general idea, if there's a public header somewhere, I'd prefer those handled via a .cdecl like the rest of our FFI stuff.